<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManager</class>
 <widget class="QWidget" name="OrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>

   <!-- 顶部筛选栏（与FlightManager查询栏风格对齐） -->
   <item>
    <widget class="QWidget" name="widgetQuery" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblOrderStatus">
        <property name="text">
         <string>订单状态:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxOrderStatus">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>全部订单</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>已支付</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>已取消</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblOrderDate">
        <property name="text">
         <string>下单日期:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="deOrderDate">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFlightNum">
        <property name="text">
         <string>航班号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFlightNum">
        <property name="placeholderText">
         <string>例如：CA1234</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #1E88E5;
    color: white;
    border-radius: 4px;
    padding: 5px 15px;
    border: none;
}
QPushButton:hover {
    background-color: #1976D2;
}
QPushButton:pressed {
    background-color: #0D47A1;
}</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Query">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 订单表格（与FlightManager表格风格对齐） -->
   <item>
    <widget class="QTableWidget" name="twOrderList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>订单ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>航班号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发城市</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达城市</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>座位号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单状态</string>
      </property>
     </column>
    </widget>
   </item>

   <!-- 订单详情栏（与FlightManager详情栏风格对齐） -->
   <item>
    <widget class="QGroupBox" name="gbOrderDetails">
     <property name="title">
      <string>订单详情信息</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblTitleOrderNo">
        <property name="styleSheet">
         <string notr="true">color: #666;</string>
        </property>
        <property name="text">
         <string>订单号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="valOrderNo">
        <property name="font">
         <font>
          <bold>true</bold>
          <weight>75</weight>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblTitleFlight">
        <property name="styleSheet">
         <string notr="true">color: #666;</string>
        </property>
        <property name="text">
         <string>航班号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="valFlight">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTitleDepart">
        <property name="styleSheet">
         <string notr="true">color: #666;</string>
        </property>
        <property name="text">
         <string>出发时间：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="valDepart">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblTitleSeat">
        <property name="styleSheet">
         <string notr="true">color: #666;</string>
        </property>
        <property name="text">
         <string>座位号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="valSeat">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 底部操作栏（新增独立取消订单按钮，样式与航班界面删除按钮一致） -->
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <!-- 新增：独立取消订单按钮 -->
      <item>
       <widget class="QPushButton" name="btnCancelOrder">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #F44336;
    color: white;
    border-radius: 4px;
    padding: 8px 20px;
    font-weight: bold;
}
QPushButton:disabled {
    background-color: #EF9A9A;
}
QPushButton:hover:enabled {
    background-color: #D32F2F;
}</string>
        </property>
        <property name="text">
         <string>取消订单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnViewDetail">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFC107; color: black; border-radius: 4px; padding: 6px;</string>
        </property>
        <property name="text">
         <string>查看详情</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>返回主页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
